usr/src/man/man1m/zpool.1m

```diff
@@ -1342,6 +1382,72 @@ Stop scrubbing.
 .El
 .It Xo
 .Nm
+.Cm trim
+.Op Fl r Ar rate | Fl s
+.Ar pool Ns ...
+.Xc
+Initiates a manual TRIM operation on all of the free space of a pool.
+This informs the underlying storage devices of all of the blocks that
+the pool no longer considers allocated, thus allowing thinly provisioned
+storage devices to reclaim them. Please note that this collects all
+space marked as "freed" in the pool immediately and doesn't wait the
+.Nm zfs_txgs_per_trim
+delay as automatic TRIM does. Hence, this can limit pool corruption recovery
```

Comment:
Not sure if removing this sentence is really the right thing.  As a ZFS user I would definitively like to know how the various commands and tunables interact. 
At least it should still be mentioned, that a manual trim temporarily renders the zpool import -F, -X and -T options unusable.

Maybe something like (I don't know troff/man syntax):
"Please note that this collects **all** space not used by active data, including recently freed regions normally postponed for several minutes to enhance pool recovery (zpool import option F, X and T)"